__tests__/pages/loading.test.tsx

```diff
@@ -270,7 +270,7 @@ it("renders correctly when online", async () => {
               <dd
                 className="govuk-summary-list__value lbh-summary-list__value"
               >
-                TestTenant1, TestTenant2
+                
```

Comment:
We should probably update the test instead. It would be good to include both cases in the data.